Notify/Notify/Notify/ViewModels/Popups/EditFriendPopupPage.cs

```diff
@@ -0,0 +1,88 @@
+using Rg.Plugins.Popup.Pages;
+using Xamarin.Forms;
+
+namespace Notify.ViewModels.Popups
+{
+    public class EditFriendPopupPage : PopupPage
+    {
+        public Picker LocationPicker { get; set; }
+        public Picker TimePicker { get; set; }
+        public Picker DynamicPicker { get; set; }
```

Comment:
Add "Permission" in the middle of each of them (for instance LocationPermissionPicker).